src/main/java/com/hyperwallet/clientsdk/util/HyperwalletEncryption.java

```diff
@@ -189,6 +192,15 @@ private JWKSet loadKeySet(String keySetLocation) throws IOException, ParseExcept
             checkKeySetLocationIsFile(keySetLocation);
             return JWKSet.load(new File(keySetLocation));
         }
+
+        if (usesProxy()) {
+            if (proxyUsername != null && proxyPassword != null) {
+                Authenticator authenticator = new MultipartRequest.DefaultPasswordAuthenticator(
```

Comment:
Minor: We should move this class (DefaultPasswordAuthenticator) out of MultipartRequest if it's used in more than one place

Reply:
Agreed, is there a better place we can host it?